mozilla-release/mobile/android/base/AppConstants.java.in

```diff
@@ -342,5 +342,6 @@ public class AppConstants {
     /* Cliqz Start */
     public static final String CLIQZ_SEARCH_VERSION="@CLIQZ_SEARCH_VERSION@";
     public static final String CLIQZ_PRIVACY_VERSION="@CLIQZ_PRIVACY_VERSION@";
+    public static final String CLIQZ_CHANNEL="@CLIQZ_CHANNEL@";
```

Comment:
I think you also need to add the constant here, I'm not sure, but I don't think it's autogenerated
https://github.com/cliqz-oss/cliqz-android/blob/master/mozilla-release/toolkit/modules/AppConstants.jsm

Reply:
Done. Please check if it works out for you.

Reply:
works